extensions/wikia/CommunityPage/models/CommunityPageSpecialInsightsModel.class.php

```diff
@@ -40,14 +51,16 @@ public function getInsightsModules() {
 
 	/**
 	 * @param string $type type of module we want to build.
-	 * @param string $sortingType define how data should be sorted (@see InsightsSorting::$sorting)
+	 * @param array $config array with variables that determine
+	 *  - should display page views on list
+	 * 	- how data should be sorted (@see InsightsSorting::$sorting)
 	 * @return array Insight Module
 	 */
-	private function getInsightModule( $type, $sortingType = self::INSIGHTS_MODULE_SORT_TYPE ) {
+	private function getInsightModule( $type, $config ) {
 		$insightPages = $this->insightsService->getInsightPages(
 			$type,
 			self::INSIGHTS_MODULE_ITEMS,
-			$sortingType
+			$config['sortingType']
```

Comment:
or if it would add too much complexity maybe easier would be to choose more unique keys and leave it this way as I see that we're already calling array properties directly in many places.... :/